nnpdf_data/nnpdf_data/commondata/BCDMS_NC_NOTFIXED_D/metadata.yaml

```diff
@@ -77,3 +77,58 @@ implemented_observables:
       experiment: DEUTERON
   data_central: data_legacy_EM-F2.yaml
   ported_from: BCDMSD
+- observable_name: EM-F2-HEPDATA
+  observable:
+    description: Deep Inelastic Scattering F2 on deuteron target.
+    label: BCDMS $d$
+    units: ''
+  process_type: DIS_NC
+  tables: [2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23]
+  npoints: []
+  ndata: 159
+  data_uncertainties: []
+  data_central: data_rqcd.yaml
+  kinematic_coverage:
+    - x
+    - Q2
```

Comment:
Up to now we were using `Q`, this is one of the magic things that plotoptions was doing, the `k2` (which was indeed `Q2`) was being converted to `Q`.

So you have two options, if you leave it like this all new plots for DIS will have `Q2` instead of `Q`, i.e., 

![image](https://github.com/user-attachments/assets/3ec4a41d-033f-41cb-89d3-0e4bb7d6e0f6)

will become

![image](https://github.com/user-attachments/assets/ea1609e4-ca1f-458d-a790-163a9ea61488)

or you need to change all `Q2` to `Q` and do `sqrt` to the `kinematics` file.

I think the second option is better since that way plots don't change and are directly comparable, but while in other cases I have a strong opinion about it, DIS is the one case where I'd be fine with using Q2 in the plots...

Choose your poison.

Reply:
If the plots in Q2 are still readable (as the one you posted) I guess that's the quickest solution and then, if needed, one can convert them in Q in a subsequent PR (together with the collider DIS data, which should have the same problem, right?)

Reply:
Yes, but the collider DIS is still to be finished so 🤷‍♂️

But I'm ok with keeping it Q^2.

Btw, the fktables for the `hepdata` observables exist in some theory or are still to be done?

Reply:
To be done...

Reply:
Then I'll run a few more checks and merge. Thanks.